cfme/base/__init__.py

```diff
@@ -65,25 +85,37 @@ def collect_logs(self):
 
     @property
     def zone(self):
-        server_res = self.appliance.rest_api.collections.servers.find_by(id=self.sid)
-        server, = server_res
+        server, = self.appliance.rest_api.collections.servers.find_by(id=self.sid)
         server.reload(attributes=['zone'])
         zone = server.zone
         zone_obj = self.appliance.collections.zones.instantiate(
-            name=zone['name'], description=zone['description'], id=zone['id']
+            name=zone['name'],
+            description=zone['description'],
+            id=zone['id']
         )
         return zone_obj
 
     @property
     def slave_servers(self):
         return self.zone.collections.servers.filter({'slave': True}).all()
 
+    @property
+    def is_slave(self):
+        return self in self.slave_servers
+
     @property
     def _api_settings_url(self):
         return '/'.join(
             [self.appliance.rest_api.collections.servers._href, str(self.sid), 'settings']
         )
 
+    @property
+    def rest_api_entity(self):
+        try:
+            return self.appliance.rest_api.collections.servers.get(id=self.sid)
+        except ValueError:
+            raise RestLookupError
```

Comment:
No message here? Like you have at https://github.com/ManageIQ/integration_tests/pull/9382/files#diff-a7dcf56bdad11918a6877cbbf1a10059R158

Reply:
I can add one here as well.